src/points/PointsDiscoverCard.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleProp, StyleSheet, Text, View, ViewStyle } from 'react-native'
+import { PointsEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import Touchable from 'src/components/Touchable'
+import LogoHeart from 'src/icons/LogoHeart'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { pointsBalanceSelector, pointsIntroHasBeenDismissedSelector } from 'src/points/selectors'
+import { useSelector } from 'src/redux/hooks'
+import { getFeatureGate } from 'src/statsig'
+import { StatsigFeatureGates } from 'src/statsig/types'
+import { Colors } from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+
+interface Props {
+  style?: StyleProp<ViewStyle>
+}
+
+export default function PointsDiscoverCard({ style }: Props) {
+  const showPoints = getFeatureGate(StatsigFeatureGates.SHOW_POINTS)
+
+  const { t } = useTranslation()
+  const pointsBalance = useSelector(pointsBalanceSelector)
+  const pointsIntroHasBeenDismissed = useSelector(pointsIntroHasBeenDismissedSelector)
+
+  const handlePress = () => {
+    ValoraAnalytics.track(PointsEvents.points_discover_press)
+    if (pointsIntroHasBeenDismissed) {
+      navigate(Screens.PointsHome)
+    } else {
+      navigate(Screens.PointsIntro)
+    }
+  }
+
+  if (!showPoints) {
+    return null
+  }
+
+  return (
+    <Touchable onPress={handlePress} testID="PointsDiscoverCard">
```

Comment:
nit: i think you need to add a `borderRadius` prop here to ensure that the highlighted part on press on android does not exceed the rounded borders

Reply:
great call! thanks! correct ripple handling required small styles/layout refactoring as well 😅